src/dataset-builder/DatasetBuilder.test.ts

```diff
@@ -258,7 +258,11 @@ describe('DatasetBuilder', () => {
     const user = userEvent.setup();
     await initializeValidDatasetRequest(user);
     // Assert
-    expect(await screen.findByText('100 Participants in this dataset')).toBeTruthy();
+    let participantText;
+    await waitFor(() => {
+      participantText = screen.findByText('100 Participants in this dataset');
+    });
```

Comment:
I might be wrong but I _think_ that you can do
```suggestion
    const participantText = await waitFor(() => (screen.findByText('100 Participants in this dataset'));
```
since `waitFor` returns a promise with the value I think

Reply:
I tried this too. I get the error that waitFor timed out. I think maybe I need to re-render the page since it is not on the initial screen, just the spinner is. Any ideas?

Reply:
Does something need to trigger the mocked fetch?

Reply:
thank you for your help! I wasn't mocking the call!